### PR TITLE
[14.0][MIG] purchase_isolated_rfq

### PR DESCRIPTION
**Pull request by SurachetS**

### CODE REVIEW

File: purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        self.assertEqual(po.partner_id, self.partner)
+        self.assertEqual(po.quote_id, self.rfq)
+        # Exceptions Case
+        with self.assertRaises(UserError) as e:
+            po.action_convert_to_order()
+        error_message = "Only quotation can convert to order"
+        self.assertEqual(e.exception.args[0], error_message)
+
+    def test_uninstall_hook(self):
+        """Test if related actions are removed when mass editing
```
Comment (kittiu) [inline purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py:38]: Why mentioned about mass editing? Can you validate the descriptoin again.

File: purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        record is uninstalled."""
```
Comment (kittiu) [inline purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py]:
```suggestion
        """Test if related actions are reset to state before this module was installed"""
```

**Review by kittiu — COMMENTED** [review]
Please rebase and push again, to test with Travis.

**Comment by kittiu** [comment]
@SurachetS please rebase.

File: purchase_isolated_rfq/models/purchase_order.py [hunk 3]
```diff
@@ -33,7 +33,6 @@ class PurchaseOrder(models.Model):
         ],
         string="RFQ Status",
         readonly=True,
-        related="state",
```
Comment (kittiu) [inline purchase_isolated_rfq/models/purchase_order.py]: @SurachetS Why remove this?
Note: I checked form v13, this is intended to be related field with store=False.

Note, we have new field rfq_state, but no value in it. And so, when I create a new RFQ, the state is not displayed.

image

**Review by kittiu — APPROVED** [review]
LGTM

**Comment by Saran440** [comment]
@SurachetS travis is red. could you fix it?

**Review by Saran440 — APPROVED** [review]
No major change :+1: 

Please, fix red travis

**Pull request merged**